dkron/job.go

```diff
@@ -303,10 +305,44 @@ func (j *Job) GetTimeLocation() *time.Location {
 	return loc
 }
 
+// scheduleHash replaces H in the cron spec by a value derived from job Name
+// such as "0 0 H * * *"
+func (j *Job) scheduleHash() string {
+	spec := j.Schedule
+	if strings.Contains(spec, "H") && strings.Count(strings.TrimSpace(spec), " ") == 5 {
+		h := 0
+		for _, c := range j.Name {
+			h += int(c)
+		}
+		parts := strings.Split(spec, " ")
+		for index, part := range parts {
+			if strings.Contains(part, "H") {
```

Comment:
It is intended yes, not because of `*/H` but for `H/5` (every 5min starting at `H`).
Not sure if `*/H` would be of any use, but I don't think it's worth actively excluding it as `*/55` (every 55minutes) is still valid cron expression 🤷 
What do you think?